_posts/2017-02-21-release-3-0-0.markdown

```diff
@@ -0,0 +1,170 @@
+---
+layout: post
+title: "Release 3.0.0"
+date: 2017-02-21 14:00:00
+author: "@jubianchi"
+categories: release
+---
+
+We are proud to announce the availability of atoum [3.0.0][milestone]!
+
```

Comment:
Here I will set, because it's very important, a note about php 7 compatibility (the void option ;)) and also that now we will support php 5.6 >= and not php 5.3 >= anymore